rules/mapping.go

```diff
@@ -69,6 +71,34 @@ func newMappingRuleSnapshot(
 	}, nil
 }
 
+type mappingRuleSnapshotJSON struct {
+	Name         string            `json:"name"`
+	Tombstoned   bool              `json:"tombstoned"`
+	CutoverNanos int64             `json:"cutoverNanos"`
+	TagFilters   map[string]string `json:"filters"`
+	Policies     []policy.Policy   `json:"policies"`
+}
+
+func newMappingRuleSnapshotJSON(mrs mappingRuleSnapshot) mappingRuleSnapshotJSON {
+	return mappingRuleSnapshotJSON{
+		Name:         mrs.name,
+		Tombstoned:   mrs.tombstoned,
+		CutoverNanos: mrs.cutoverNanos,
+		TagFilters:   mrs.rawFilters,
+		Policies:     mrs.policies,
+	}
+}
+
+func (mrsj mappingRuleSnapshotJSON) mappingRuleSnapshot() mappingRuleSnapshot {
+	return mappingRuleSnapshot{
```

Comment:
Perhaps better to add a `newMappingRuleSnapshot` ctor taking in these params and returning a `mappingRuleSnapshot` and reuse the ctor here and in other places?